cluster/gke/config-test.sh

```diff
@@ -20,6 +20,7 @@ NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
 IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-container_vm}"
 ENABLE_KUBERNETES_ALPHA="${KUBE_GKE_ENABLE_KUBERNETES_ALPHA:-}"
+NODE_OS_DISTRIBUTION=gci
```

Comment:
I kinda wish there were a better way to detect this than relying on an environment variable/flag to be set properly...

is `NODE_OS_DISTRIBUTION` used by any cluster providers besides GCE/GKE?